deployment/k8s/gke-kube-dashboard-rbac.yml

```diff
@@ -1,12 +1,13 @@
-apiVersion: rbac.authorization.k8s.io/v1beta1
+# When enable RBAC on GKE, Dashboard is not accessible without bellow CRB
```

Comment:
why did you done a `git mv` of `cluster-role.binding.yaml` to `deployment/k8s/gke-kube-dashboard-rbac.yml`
I don't understand why we have something specific to GKE here?

Reply:
It's not a moved file, but an additional CRB file. I'm using GKE K8s cluster. When you enable RBAC on GKE, you also need to create a new CRB to be able to access K8s Dashboard. 
Maybe should move it to README.md or into some `helper/gke` folder